### PR TITLE
[Unitary[ Create game_design.md

### PR DESCRIPTION
**Pull request by dstrain115**

### CODE REVIEW

File: docs/unitary/game_design.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
+# Game Design
```
Comment (cognigami) [inline docs/unitary/game_design.md:1]: Remove space between # and Game.

File: docs/unitary/game_design.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+This guide explains principles that can help you decide how to implement quantum effects in your game idea.
+
+## State
```
Comment (cognigami) [inline docs/unitary/game_design.md:5]: Remove space.

File: docs/unitary/game_design.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+   * By applying a `Split` effect.
+   * By applying a `Superposition` effect.
+   * By using the `effect_fraction` argument when you call an effect that supports it.
+
+---
+You can apply this principle to any property of the door, such as the destination beyond the door, whether it is locked, and so on.
+
+---
+
+## Entanglement
```
Comment (cognigami) [inline docs/unitary/game_design.md:29]: Remove space.

File: docs/unitary/game_design.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+As your quantum state grows, entangled objects can prompt non-obvious outcomes for your game objects.
+
+You can create entanglement when you apply an effect conditionally between quantum objects, by using the `quantum_if()` method.
+
+---
+You can design a game mechanic that links properties of the door to the player’s actions in the game. For example, the location or destination of the door might be linked to other doors or rooms that the player has visited, or to a key that the player uses to unlock the door.
+
+---
+
+## Interference
```
Comment (cognigami) [inline docs/unitary/game_design.md:42]: Remove space.

File: docs/unitary/game_design.md [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+   * Your objects must have phase.
+
+You can create the opportunity for interference effects by using the `Phase` effect, or the `effect_fraction` argument when you use a `QuantumEffect` that accepts the argument.
+
+---
+The location or destination of a door might depend on previous engagements with the door. 
+
+---
+
+## Measurement
```
Comment (cognigami) [inline docs/unitary/game_design.md:60]: Remove space.

File: docs/unitary/game_design.md [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+Use the `pop` method on your `QuantumWorld` object to retrieve the classical state of your quantum objects.
+
+---
+If you need to choose a destination for the player who walks through the door, you might need to measure the quantum state. 
+
+Alternatively, you could enable your player to play in superposition if the door could send the player to multiple possible destinations. In this case, the player becomes entangled with the state of the door.
+
+---
+
+## Simplicity
```
Comment (cognigami) [inline docs/unitary/game_design.md:75]: Remove space.

File: docs/unitary/game_design.md [hunk 7]
```diff
@@ -0,0 +1,96 @@
...
+
+Keep the quantum state of your game simple. A smaller quantum state makes your game easier for the player to understand. A smaller quantum state also improves your ability to run your game on a near-term quantum computer.
+
+Simplicity is a process. Use this opportunity to think again about the division between the quantum state and the classical state of your game.
+
+## Composability
```
Comment (cognigami) [inline docs/unitary/game_design.md:81]: Remove space.

File: docs/unitary/game_design.md [hunk 8]
```diff
@@ -0,0 +1,96 @@
...
+
+The Unitary library allows for any quantum object to have up to four states. This attribute balances the limits of current classical computing and quantum computing hardware with expressivity and creativity.
+
+The current set of quantum games, like Quantum Chess and Tiq Taq Toe, generate engagement through the interaction of quantum objects rather than quantum objects with a multitude of states.
+
+When you design your quantum game, think about expanding the set of probable outcomes through the interaction of quantum objects, rather than adding more states to your object. In some cases, you might need to think differently about the data that you track about your objects. 
+
+For ideas about working with the four-state limitation, see the following section.
+
+## Example: Quantum Chess
```
Comment (cognigami) [inline docs/unitary/game_design.md:91]: Remove space.

**Review by cognigami — CHANGES_REQUESTED** [review]

File: docs/unitary/game_design.md (hunk 3 above)
Comment (dstrain115) [reply docs/unitary/game_design.md:29]: Same.

File: docs/unitary/game_design.md (hunk 5 above)
Comment (dstrain115) [reply docs/unitary/game_design.md:60]: Same.

File: docs/unitary/game_design.md (hunk 6 above)
Comment (dstrain115) [reply docs/unitary/game_design.md:75]: Same.

File: docs/unitary/game_design.md (hunk 7 above)
Comment (dstrain115) [reply docs/unitary/game_design.md:81]: Same.

File: docs/unitary/game_design.md (hunk 8 above)
Comment (dstrain115) [reply docs/unitary/game_design.md:91]: Same.

**Review by cognigami — APPROVED** [review]

**Pull request merged**